.ci/lint.R

```diff
@@ -0,0 +1,72 @@
+library(lintr)
+
+args <- commandArgs(
+    trailingOnly = TRUE
+)
+SOURCE_DIR <- args[[1]]
+
+FILES_TO_LINT <- list.files(
+    path = SOURCE_DIR
+    , pattern = "\\.r$"
+    , all.files = TRUE
+    , ignore.case = TRUE
+    , full.names = TRUE
+    , recursive = TRUE
+    , include.dirs = FALSE
+)
+
+# Some linters from the lintr package have not made it to CRAN yet
+# We build lintr from source to address that.
```

Comment:
could you remove these comments? I realize I left them behind in the code I pointed you to and they aren't relevant. Sorry!